scvi/inference/inference.py

```diff
@@ -47,7 +50,10 @@ def loss(self, tensors):
         return loss
 
     def on_epoch_begin(self):
-        self.kl_weight = self.kl if self.kl is not None else min(1, self.epoch / 400)  # self.n_epochs)
+        if self.n_epochs_kl_warmup is not None:
+            self.kl_weight = min(1, self.epoch / self.n_epochs_kl_warmup)
+        else:
```

Comment:
I know that certain datasets really need a different warmup to get good clustering results. Is there a way to keep the same default than before ?

Reply:
Yes, basically just make the default value 400, which I will do.